file_format.md

```diff
@@ -68,7 +68,17 @@ JSON files accepted by the simularium-viewer contain the following data in JSON
 {
     // trajectory info
     "trajectoryInfo" : {
-        "version" : 2,
+        "version" : 3,
+        // model metadata
+        "trajectoryTitle" : "Fast diffusion",
+        "modelInfo" : {
+            "title" : "SARS-CoV-2 Dynamics in Human Lung Epithelium"
+            "version" : 4.1
+            "authors" : "Michael Getz et al"
+            "description" : "A PhysiCell model of SARS-CoV-2 dynamics in human lung epithelium."
+            "doi" : "10.1101/2020.04.02.019075"
+            "sourceCodeUrl" : "https://github.com/pc4covid19/pc4covid19"
+        },
```

Comment:
Should this and the geometry fields be updated in the above bulleted list descriptions too? (lines 4-64)